dataactvalidator/scripts/loaderUtils.py

```diff
@@ -13,11 +13,7 @@ def padFunction(self, field, padTo, keepNull):
         """Pads field to specified length."""
         if isnull(field) or not str(field).strip():
             if keepNull:
```

Comment:
Yep, we need it. You're right that it's gone from the SF-133 load function, but it's still used in `loadProgramActivity.`